deployment/roles/ui/tasks/main.yml

```diff
@@ -0,0 +1,3 @@
+- include_tasks: pre_config.yml
+- include_tasks: jumpbox.yml
+- include_tasks: servinstall.yml
```

Comment:
does it make sense to have a task `logging.yml` especially for the standalone UI deployment?

Reply:
The PR is losing on clarity - I created https://github.com/poanetwork/tokenbridge/issues/128 to do this in a subsequent PR if that's OK with you.

Reply:
IMO it should be done in this PR, currently docker logs will keep accumulating eating up disk space.

Reply:
OK. But please address it next